spec/unit/mutant/matcher/methods/instance_spec.rb

```diff
@@ -96,9 +96,11 @@ def apply
     it 'warns about degnerate interfacew' do
       apply
 
+      method_name = :foo
+
       exception =
         begin
-          scope.instance_method(:foo)
+          scope.instance_method(method_name)
```

Comment:
This allows to replicate exactly the same exception we can match against, since ruby 3.1 does source location reflection in exception messages right now. `instance_method(:foo)` would fail this test.